src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/Expected/MixedMode_x64_VS2015_Default.exe.sarif

```diff
@@ -988,20 +988,20 @@
               "id": "BA6001",
               "name": "DisableIncrementalLinkingInReleaseBuilds",
               "fullDescription": {
-                "text": "Incremental linking support increases binary size and can reduce runtime performance. Fully optimized release builds should not specify incremental linking."
+                "text": "Incremental linking support increases binary size and can reduce runtime performance. The support for incremental linking adds padding and other overhead to support the ability to modify a binary without a full link.  The use of incrementally linked binaries may reduce the level of determinism because previous compilations will have lingering effects on subsequent compilations.  Fully optimized release builds should not specify incremental linking."
```

Comment:
>"text": "Incremental [](http://example.com/codeflow?start=16&length=20)

I re-ran the baseline to see if my change affect baselines, all the changes are related to previous change `BA6001`.

Reply:
Sorry about that one.  Somehow this staleness didn't break anything so I didn't notice it when I updated the strings based on PR feedback.  I was seeing these updates too when working on some additional rules and updating the baselines accordingly.

Reply:
no worries at all I just call it out to save reviewer's when see this diff.

With no enforcement in the build and test no one will remember to run re-baseline.
(
My preference is to change the re-baseline test to strict compare so that we are forced to re-run baseline and review the changes caused, expectedly and unexpectedly :) 
)

Reply:
Shaopeng, if you haven't already, please create an Issue to track this suggestion.

Reply:
Done. 
https://github.com/microsoft/binskim/issues/694